src/System.Data.SqlClient/src/System/Data/Common/AdapterUtil.cs

```diff
@@ -827,69 +828,14 @@ static internal Exception TransactionZombied(DbTransaction obj)
-        internal const string DeriveParameters = "DeriveParameters";
-        internal const string EndExecuteNonQuery = "EndExecuteNonQuery";
         internal const string EndExecuteReader = "EndExecuteReader";
         internal const string EndExecuteXmlReader = "EndExecuteXmlReader";
-        internal const string ExecuteReader = "ExecuteReader";
-        internal const string ExecuteRow = "ExecuteRow";
-        internal const string ExecuteNonQuery = "ExecuteNonQuery";
-        internal const string ExecuteScalar = "ExecuteScalar";
-        internal const string ExecuteSqlScalar = "ExecuteSqlScalar";
-        internal const string ExecuteXmlReader = "ExecuteXmlReader";
-        internal const string Fill = "Fill";
-        internal const string FillPage = "FillPage";
-        internal const string FillSchema = "FillSchema";
-        internal const string GetBytes = "GetBytes";
-        internal const string GetChars = "GetChars";
-        internal const string GetOleDbSchemaTable = "GetOleDbSchemaTable";
-        internal const string GetProperties = "GetProperties";
-        internal const string GetSchema = "GetSchema";
-        internal const string GetSchemaTable = "GetSchemaTable";
-        internal const string GetServerTransactionLevel = "GetServerTransactionLevel";
-        internal const string Insert = "Insert";
-        internal const string Open = "Open";
         internal const string Parameter = "Parameter";
-        internal const string ParameterBuffer = "buffer";
-        internal const string ParameterCount = "count";
-        internal const string ParameterDestinationType = "destinationType";
-        internal const string ParameterIndex = "index";
         internal const string ParameterName = "ParameterName";
-        internal const string ParameterOffset = "offset";
         internal const string ParameterSetPosition = "set_Position";
-        internal const string ParameterService = "Service";
-        internal const string ParameterTimeout = "Timeout";
-        internal const string ParameterUserData = "UserData";
-        internal const string Prepare = "Prepare";
-        internal const string QuoteIdentifier = "QuoteIdentifier";
-        internal const string Read = "Read";
-        internal const string ReadAsync = "ReadAsync";
-        internal const string Remove = "Remove";
-        internal const string RollbackTransaction = "RollbackTransaction";
-        internal const string SaveTransaction = "SaveTransaction";
-        internal const string SetProperties = "SetProperties";
-        internal const string SourceColumn = "SourceColumn";
-        internal const string SourceVersion = "SourceVersion";
-        internal const string SourceTable = "SourceTable";
-        internal const string UnquoteIdentifier = "UnquoteIdentifier";
-        internal const string Update = "Update";
-        internal const string UpdateCommand = "UpdateCommand";
-        internal const string UpdateRows = "UpdateRows";
```

Comment:
All the removed constants above were either dead code before this PR or became dead code as part of this PR by switching uses to `nameof()` or `[CallerMemberName]`.